### DISCUSSION TITLE
Mapping EIA 860 generators to FERC Form 1 generators

### DISCUSSION THREAD

**Discussion by cody-w** [catalyst-cooperative/pudl#1691] [Help Me!] [answered]
Hi there- I've worked with EIA 860 and FERC Form 1 data quite a lot but only recently came across the PUDL project.

I have a very specific question-- has the PUDL project ever mapped the names of plants in FERC Form 1 to the EIA IDs of these plants in EIA 860? 

In FERC Form 1, this would be the _plant_name_ field in the [f1_gnrt_plant](https://data.catalyst.coop/ferc1/f1_gnrt_plant) dataset.

In EIA 860, this would be the _plant_id_eia_ field in the [generators_eia860](https://data.catalyst.coop/pudl/generators_eia860) dataset.

I am looking for a way to merge data on plant capex and maintenance costs from FERC Form 1 with other useful info provided by EIA 860. The issue is plant names can be typed or reported in different ways between both datasets.

I am interested if the PUDL project has developed this kind of dictionary before? If not, I will likely develop one on my own and would be happy to provide it open-access. 

Thanks!

**Comment by zaneselvans** [comment, marked as answer]
Yes, we've done a tone of work to connect the FERC 1 and EIA plants, specifically to be able to bring the CapEx/OpEx numbers from FERC together with the more detailed fuel costs and generator-level ownership information on the EIA side. See the [RMI FERC1-EIA project](https://github.com/catalyst-cooperative/rmi-ferc1-eia/) repo. @cmgosnell and @aesharpe would be the ones to talk to. There's a lot more details on the capital & operating costs for the `f1_steam` (large plants) table than for `f1_gnrt_plant` (small plants) though (and also almost all the capital and generation and emissions are associated with the `f1_steam` plants). We've also compiled a bunch of detailed information from state-level depreciation studies, and are working with the Chu Data Lab at Georgia Tech to try and automate some of this record linkage.

**Reply by cody-w** [reply]
Thanks so much @zaneselvans. This is really helpful and saves me a chunk of time. Looking into it now and will reach out to those folks with questions.
